packages/imperative/src/config/__tests__/Config.secure.unit.test.ts

```diff
@@ -167,6 +167,21 @@ describe("Config secure tests", () => {
         ]);
     });
 
+    it("should list all secure fields for a layer", async () => {
+        jest.spyOn(Config, "search").mockReturnValue(projectConfigPath);
+        jest.spyOn(fs, "existsSync")
+            .mockReturnValueOnce(false)     // Project user layer
+            .mockReturnValueOnce(true)      // Project layer
+            .mockReturnValueOnce(false)     // User layer
+            .mockReturnValueOnce(false);    // Global layer
+        jest.spyOn(fs, "readFileSync");
+        const config = await Config.load(MY_APP);
+        config.mSecure = secureConfigs;
+        expect(config.api.secure.secureFieldsForLayer(projectConfigPath)).toEqual({ [securePropPath]: "area51" });
+        expect(config.api.secure.secureFieldsForLayer(projectUserConfigPath)).toEqual(null);
+        config.mSecure = {};
```

Comment:
I know this is being a little picky, but is this line necessary for the locally scoped `config` variable?